client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -91,6 +94,42 @@ export function getEnhancedTasks(
 						},
 					};
 					break;
+				case 'videopress_setup':
+					taskData = {
+						title: translate( 'Set up your Video Site' ),
+					};
+					break;
+				case 'videopress_upload':
+					taskData = {
+						title: translate( 'Upload your first video' ),
+						actionUrl: `/site-editor/${ siteSlug }`,
+						isCompleted: videoPressUploadCompleted,
+					};
+					break;
+				case 'videopress_launched':
+					taskData = {
+						title: translate( 'Launch Video Site' ),
```

Comment:
```suggestion
						title: translate( 'Launch Video site' ),
```
Another "not entirely sure about" comment, even though it looks odd, should `site` be lowercased?